src/cript/nodes/primary_nodes/data.py

```diff
@@ -1,9 +1,307 @@
+from dataclasses import dataclass, field, replace
+from typing import Any, List
+
+# from cript import File, Process, Computation, ComputationalProcess, Material, Citation
 from cript.nodes.primary_nodes.primary_base_node import PrimaryBaseNode
 
 
 class Data(PrimaryBaseNode):
     """
     Data node
+    [Data node](https://pubs.acs.org/doi/suppl/10.1021/acscentsci.3c00011/suppl_file/oc3c00011_si_001.pdf#page=13)
     """
 
-    pass
+    @dataclass(frozen=True)
+    class JsonAttributes(PrimaryBaseNode.JsonAttributes):
+        """
+        all Data attributes
+        """
+
+        type: str = ""
+        # TODO add proper typing in future, using Any for now to avoid circular import error
+        files: List[Any] = field(default_factory=list)
+        sample_preperation: Any = field(default_factory=list)
+        computations: List[Any] = field(default_factory=list)
+        computational_process: Any = field(default_factory=list)
+        materials: List[Any] = field(default_factory=list)
+        processes: List[Any] = field(default_factory=list)
+        citations: List[Any] = field(default_factory=list)
+
+    _json_attrs: JsonAttributes = JsonAttributes()
+
+    def __init__(
+        self,
+        name: str,
+        type: str,
+        files: List[Any],
+        sample_preperation: Any = None,
+        computations: List[Any] = None,
+        computational_process: Any = None,
+        materials: List[Any] = None,
+        processes: List[Any] = None,
+        citations: List[Any] = None,
+        notes: str = "",
+        **kwargs
+    ):
+        super().__init__(node="Data", name=name, notes=notes)
+
+        if files is None:
+            files = []
```

Comment:
Use some loop for this for cleanliness

Reply:
yeah I agree! I thought of making some sort of function for it, but that code clean up, refactoring, and optimization might need to come later so we can get this code out soon.